ui/components/app/hold-to-reveal/index.scss

```diff
@@ -0,0 +1,41 @@
+.btn--rounded.btn-primary {
+  width: 190px;
+  height: 40px;
+  padding: 6px 13px 6px 9px;
+
+  &:active {
+    background-color: var(--Blue-500) !important;
+  }
+}
```

Comment:
This will change all components with the classes `.btn--rounded.btn-primary` I would suggest giving the component a root class and updating those styles using the root class.